lib/plugins/random-messages/index.js

```diff
@@ -21,15 +21,27 @@ const unknownMessageReplies = [
 
 async function pluginRandomMessages (client) {
   client.on('message', msg => {
-    for (const word of triggerWords) {
-      if (msg.content.toLowerCase().includes(word)) {
-        return msg.reply(getRandomArrayItem(knownMessageReplies))
-      }
-    }
-    msg.reply(getRandomArrayItem(unknownMessageReplies))
+    const msgLower = msg.toLowerCase()
+    const containsTriggerWord = triggerWords.find(triggerWord =>
+      msgLower.includes(triggerWord)
+    )
+
+    const replies = containsTriggerWord
+      ? knownMessageReplies
+      : unknownMessageReplies
+    const reply = getRandomArrayItem(replies)
+
+    msg.reply(reply)
   })
 }
 
 pluginRandomMessages[PLUGIN_NAME] = 'Random Messages'
 
 module.exports = { pluginRandomMessages }
+
+/* for (const word of triggerWords) {
+      if (msg.content.toLowerCase().includes(word)) {
+        return msg.reply(getRandomArrayItem(knownMessageReplies))
+      }
+    }
+    msg.reply(getRandomArrayItem(unknownMessageReplies)) */
```

Comment:
Ezt vedd ki kérlek, nem rakunk be kommentelt kódot. Ha valamiért ez kell úgy is ott van az előző commitban.

Reply:
Ez véletlen maradt benne, más késő volt...